country-dashboard/src/components/Card.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+// import countries from './data/countries.json';
+
+function Card(props:any) {
```

Comment:
Don't use `function` declarations, they're not as clean. Use arrow functions, like `const Card = (...) => {...}`